dask/tests/test_distributed.py

```diff
@@ -21,6 +27,7 @@
 from dask.distributed import futures_of, wait
 from dask.highlevelgraph import HighLevelGraph, MaterializedLayer
 from dask.utils import get_named_args, tmpdir, tmpfile
+from dask.utils_test import inc
```

Comment:
Unrelated change. We already have an `inc` function in the `dask.utils_test` so no need to import it from `distributed.utils_rest`